engine/core/src/main/java/org/datacleaner/metamodel/datahub/DataHubDataSet.java

```diff
@@ -158,6 +160,10 @@ private String createEncodedUri(DataHubConnection connection, Table table) {
     }
 
     private String getQueryString(Query query, Table table) {
+        final OrderByClause orderByClause = query.getOrderByClause();
+        if(orderByClause.isEmpty()) {
```

Comment:
We are adding the ordered by now only when the select item is not a function (but a plain select of columns).